index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Mochiy+Pop+P+One&family=Poppins:wght@300&display=swap" rel="stylesheet">
+    
+</head>
+<body>
+    <header class="bright-color">
+        <h3 class="purple logo">Hello!</h3>
+        <img id="hamburger-btn" src="Union.png" alt="hamburger" width="40" height="40">
```

Comment:
- [ ] Please wrap your hamburger icon inside the `<nav>` tag so as to enhance your semantics.

Reply:
@karvel-code is it okay to wrap them inside button tag?